SoftLayer/CLI/hardware/create_options.py

```diff
@@ -81,8 +81,9 @@ def _preset_prices_table(sizes, tables):
     preset_prices_table.sortby = 'Value'
     preset_prices_table.align = 'l'
     for size in sizes:
-        preset_prices_table.add_row([size['name'], size['key'], "%.4f" % size['hourlyRecurringFee'],
-                                     "%.4f" % size['recurringFee']])
+        if ("%.4f" % size['hourlyRecurringFee'] != '0.0000') or ("%.4f" % size['recurringFee'] != '0.0000'):
```

Comment:
You can just check if `if (size['hourlyRecurringFee'] > 0) or (size['recurringFee'] > 0):`